### ISSUE TITLE
where does `craft_mlt_25k.h5` come from? 

### ISSUE THREAD

**Issue by schissmantics** [faustomorales/keras-ocr#188]
Hi, thanks for your great work which I'm referencing in my version of craft. I noticed that in the training [example](https://keras-ocr.readthedocs.io/en/latest/examples/end_to_end_training.html#) your code looks for / downloads a `craft_mlt_25k.h5`. Assuming I want to train a model from scratch without pretrained weights, after n training epochs, should I expect to get the same weights or close? If not, then how these weights were originally obtained? I also noticed that the training always starts with a loss value of 0.0something whether pretrained weights are loaded or not. This is making me suspect whether the training is actually working specially with the use of MSE as loss function which is different from the paper.

**Comment by Asad-Raza** [comment]
Hi ,
Did you get answer to the question? Im also interested in knowing the answer for this question.

Does Keras OCR use pretrained model of pyTorch CRAFT implementation?

**Comment by schissmantics** [comment]
> Hi , Did you get answer to the question? Im also interested in knowing the answer for this question.
> 
> Does Keras OCR use pretrained model of pyTorch CRAFT implementation?

Not yet but I'm assuming these weights were converted from pytorch, and clearly there's something wrong with the loss starting at a very low value.
